.github/scripts/codecheck.py

```diff
@@ -9,14 +9,14 @@
 JSONRPSEE_RE = r'\bjsonrpsee[_a-z0-9]*::'
 
 LICENSE_TEMPLATE = [
-    r'// Copyright \(c\) 202[0-9](-202[0-9])? RBB S\.r\.l',
+    r'// Copyright \(c\) 202[0-9](-202[0-9])? .+',
     r'// opensource@mintlayer\.org',
     r'// SPDX-License-Identifier: MIT',
     r'// Licensed under the MIT License;',
     r'// you may not use this file except in compliance with the License\.',
     r'// You may obtain a copy of the License at',
     r'//',
-    r'//\s+http://spdx\.org/licenses/MIT',
+    r'//\s+https://github.com/mintlayer/mintlayer-core/blob/master/LICENSE',
```

Comment:
One more point about the URL. This points to the LICENSE file in the `master` branch. In case re-licensing is needed for some reason, the URL might start pointing to a different license than what the header mentions. Should it point to particular tag or commit instead? I am not sure how likely this scenario is or what should happen if it comes up, so maybe I am off the mark completely here.

Reply:
If we re-license we'll have a load of other things to change too, `r'// SPDX-License-Identifier: MIT',` for example. What you're saying is true but given how many other things will require changing it probably saves a few minutes of time at most.